jobs/haproxy/monit

```diff
@@ -12,6 +12,8 @@ end
 -%>
 
 <%- if p("ha_proxy.enable_health_check_http") -%>
+
+<%- if not p("ha_proxy.disable_monit_health_check_http") -%>
```

Comment:
Here we need to check if `ha_proxy.enable_health_check_http` is true first and if `ha_proxy.disable_monit_health_check_http` is false second.